jpademo/src/main/java/com/likelion/jpademo/service/PostService.java

```diff
@@ -0,0 +1,54 @@
+package com.likelion.jpademo.service;
+
+import com.likelion.jpademo.dto.PostResponse;
+import com.likelion.jpademo.entity.Member;
+import com.likelion.jpademo.entity.Post;
+import com.likelion.jpademo.exception.NotFoundException;
+import com.likelion.jpademo.repository.PostRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class PostService {
+
+    private final PostRepository postRepo;
+    private final MemberService memberService;
+
+    @Transactional
+    public PostResponse create(Long memberId, String title, String content) {
+        Member member = memberService.findById(memberId);
+        Post post = Post.builder()
+                .title(title)
+                .content(content)
+                .member(member)
+                .build();
+        Post saved = postRepo.save(post);
+        return PostResponse.fromEntity(saved);
+    }
```

Comment:
`member.addPost(post);`가 있어야 양방향 연관관계가 설정되겠죠~?
```suggestion
    @Transactional
    public PostResponseDto create(Long memberId, String title, String content) {
        Member member = memberRepo.findById(memberId)
                .orElseThrow(() -> new NotFoundException("Member not found: " + memberId));
        Post post = Post.builder().title(title).content(content).member(member).build();
        member.addPost(post);
        Post saved = postRepo.save(post);
        return PostResponseDto.from(saved);
    }
```